spec.html

```diff
@@ -11868,6 +11898,13 @@ <h1>Runtime Semantics: PropertyDefinitionEvaluation</h1>
           1. Assert: _enumerable_ is *true*.
           1. Return CreateDataPropertyOrThrow(_object_, _propKey_, _propValue_).
         </emu-alg>
+        <emu-alg>
+          1. Let _exprValue_ be the result of evaluating _AssignmentExpression_.
+          1. Let _fromValue_ be GetValue(_exprValue_).
+          1. ReturnIfAbrupt(_fromValue_).
```

Comment:
This line should be removed, and the preceding `GetValue` be prefixed with a `?`